src/util/screening-selector.js

```diff
@@ -1,23 +1,23 @@
 
-import questionnaireUsAudit from '../fhir/Questionnaire-USAUDIT.json';
-import questionnaireWhoAudit from '../fhir/Questionnaire-WHOAUDIT.json';
-import questionnaireNidaQs from '../fhir/Questionnaire-NIDAQS2USAUDIT.json';
-import elmJsonUsAudit from '../cql/UsAuditLogicLibrary.json';
-import elmJsonWhoAudit from '../cql/WhoAuditLogicLibrary.json';
-import elmJsonNidaQs from '../cql/NidaQsToUsAuditLogicLibrary.json';
 import valueSetJsonUsAudit from '../cql/valueset-db.json';
 
-
-export function getScreeningInstrument() {
-  let screeningInstrument = process.env.VUE_APP_ALCOHOL_SCREENING_INSTRUMENT.toLowerCase();
-
+//dynamically load questionnaire and cql JSON
```

Comment:
@achen2401  I'd like to review the vue life cycle with you to better understand the scope here. I'm curious what this looks like if we're presenting a queue of questionnaires (I think the asbi system only does one).
For my reference: https://vuejs.org/guide/essentials/lifecycle.html#lifecycle-diagram

Reply:
@mcjustin yes, that is my understanding, too.  From what I have gathered,  the tool currently only presents one questionnaire at a time.  If we have a queue of questionnaires we will mostly likely need to refactor the code.  Also, it looks like the tool that transforms FHIR questionnaire into a SurveyJS json format also only supports **one** questionnaire.  I am looking at [here](https://github.com/asbi-cds-tools/questionnaire-to-survey/blob/751baa61f81314f7026ac5f7a5a34e03a80eeac5/fhirConversionTools.js#L8).

Reply:
Thanks for digging in Amy!

> If we have a queue of questionnaires we will mostly likely need to refactor the code

Another option could be to queue them in the EHR (launching the app for each survey), and launch for each Questionnaire. However, SoF may not be designed for that

Reply:
@ivan-c  re `Another option could be to queue them in the EHR`

I was thinking that a patient-specific CarePlan would be upstream of this scope (preferably stored in the EHR), with a PlanDefinition upstream of that. 

The asbi systems are a bit murky to me re actual implementation vs name-dropping:

No reference to "PlanDefinition" in the code, for neither the screener nor the intervention.
But, there is a json file for PlanDefinition in the *LogicFiles.zip for each of the 3 sub-audits, eg "PlanDefinition-AlcoholScreeningWhoAudit.json". 
These include a reference to "ApplyScreeningInstrument", which is defined in the CQL.

CarePlan
Only reference in the code is in asbi-screening-app/asbi-screening-app/src/cql/FHIRHelpers-4.0.1.json
eg:"name" : "{http://hl7.org/fhir}CarePlan[Status|Intent|ActivityStatus|ActivityIntent]"
No sample json files found for CarePlan